examples/get-scope-file/README.md

```diff
@@ -0,0 +1,40 @@
+# Get Sigrid scope configuration file
+
+Script that uses the Sigrid API to retrieve and dump the Sigrid [scope configuration file](https://docs.sigrid-says.com/reference/analysis-scope-configuration.html) for a system. 
+It is recommended to store the scope configuration file in your repository, named `sigrid.yaml`, and have it under version control. 
+However, you can use this example to retrieve the scope file in scenarios where it is not possible to store the scope configuration file in the repository, but you still want to retrieve it.
+
+**Note:** This script is intended for retrieving the scope configuration file for the analysis results *currently* in Sigrid.
+This script is *not* suitable for "live" editing. If you want a fast feedback loop while editing scope files, we recommend you use the [Visual Studio Code and JetBrains support for editing scope configuration files](https://docs.sigrid-says.com/reference/analysis-scope-configuration.html#editing-scope-files).
+
+## Prerequisites
+
+You will need the following to use this script.
+
+- The script requires Python 3.9+.
+- You will need a valid [API token](https://docs.sigrid-says.com/organization-integration/authentication-tokens.html) to access the [Sigrid REST API](https://docs.sigrid-says.com/integrations/sigrid-api-documentation.html).
+- Your API token should be available to the script as the environment variable `SIGRID_CI_TOKEN`.
+
+## Usage
+
+Once all prerequisites are in place, you can use the script:
+
+    ./get_sigrid_scope_file.py --customer mycustomername --system mysystemname
```

Comment:
Can we mark this as a code block for readability, and indicate which are the placeholders?
```suggestion
```
    ./get_sigrid_scope_file.py --customer <mycustomername> --system <mysystemname>
```
```

Reply:
Ah, I didn't see that it's already a code block due to 4-space indent. In other examples we also don't mark the placeholders with `< >` so we could leave that here as well.